secant/UIComponents/Buttons/NeumorphicDesignModifier.swift

```diff
@@ -0,0 +1,79 @@
+//
+//  NeumorphicDesignModifier.swift
+//  secant-testnet
+//
+//  Created by Lukáš Korba on 02/21/22.
+//
+
+import SwiftUI
+
+/// Neumorphic design is charasterictical with two shadows (light & dark) around the view
+/// Appereance in our case is influenced by two parameters:
+/// - Parameters:
+///   - colorScheme: The light is using full neumorphic design while dark is limited to soft shadow only
+///   - isPressed: When the button is pressed, there are different behaviours for light vs. dark colorScheme
+/// This design is mostly used for CircularFrame, not designed for a button (see NeumorphicButtonDesign)
+struct NeumorphicDesign: ViewModifier {
+    @Environment(\.colorScheme) var colorScheme
+    let isPressed: Bool
+    
+    init(_ isPressed: Bool) {
```

Comment:
Is this init function needed because of the colorScheme variable or is it possible to remove this and use the generated member wise initializer?

Reply:
yes, the colorScheme is requested otherwise here
```swift
self.modifier(Neumorphic(isPressed))
```